src/vaadin-details.html

```diff
@@ -147,22 +152,33 @@
           });
         }
 
+        /** @private */
         _getAriaExpanded(opened) {
           return opened ? 'true' : 'false';
         }
 
+        /** @private */
         _getAriaHidden(opened) {
           return opened ? 'false' : 'true';
         }
 
+        /** @private */
         _openedChanged(opened) {
           this._collapsible.style.maxHeight = opened ? '' : '0px';
         }
 
+        /**
+         * @param {MouseEvent!} e
```

Comment:
Move `!` to beginning for consistency.
```suggestion
         * @param {!MouseEvent} e
```

Reply:
Done.